<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Viewer</string>
  </property>
  <property name="windowIcon">
   <iconset theme="lximage-qt">
    <normaloff/>
   </iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="LxImage::ImageView" name="view">
      <property name="renderHints">
       <set>QPainter::Antialiasing|QPainter::HighQualityAntialiasing|QPainter::TextAntialiasing</set>
      </property>
      <property name="dragMode">
       <enum>QGraphicsView::ScrollHandDrag</enum>
      </property>
      <property name="transformationAnchor">
       <enum>QGraphicsView::AnchorUnderMouse</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>543</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNewWindow"/>
    <addaction name="actionOpenFile"/>
    <addaction name="actionOpenDirectory"/>
    <addaction name="actionScreenshot"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionFileProperties"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuGo">
    <property name="title">
     <string>Go</string>
    </property>
    <addaction name="actionPrevious"/>
    <addaction name="actionNext"/>
    <addaction name="actionFirst"/>
    <addaction name="actionLast"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionZoomIn"/>
    <addaction name="actionZoomOut"/>
    <addaction name="actionOriginalSize"/>
    <addaction name="actionZoomFit"/>
    <addaction name="separator"/>
    <addaction name="actionShowThumbnails"/>
    <addaction name="separator"/>
    <addaction name="actionFullScreen"/>
    <addaction name="actionSlideShow"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionRotateClockwise"/>
    <addaction name="actionRotateCounterclockwise"/>
    <addaction name="actionFlipHorizontal"/>
    <addaction name="actionFlipVertical"/>
    <addaction name="separator"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionUpload"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_View"/>
   <addaction name="menuGo"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Toolbar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPrevious"/>
   <addaction name="actionNext"/>
   <addaction name="separator"/>
   <addaction name="actionOpenFile"/>
   <addaction name="actionSaveAs"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionRotateClockwise"/>
   <addaction name="actionRotateCounterclockwise"/>
   <addaction name="actionZoomIn"/>
   <addaction name="actionZoomOut"/>
   <addaction name="actionZoomFit"/>
   <addaction name="actionOriginalSize"/>
   <addaction name="separator"/>
   <addaction name="actionSlideShow"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="icon">
    <iconset theme="help-about">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionOpenFile">
   <property name="icon">
    <iconset theme="document-open">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Open File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset theme="document-save">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset theme="document-save-as">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Save &amp;As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset theme="zoom-in">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Zoom &amp;In</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset theme="zoom-out">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Zoom &amp;Out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset theme="edit-copy">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Copy to Clipboard</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset theme="go-next">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Next File</string>
   </property>
   <property name="toolTip">
    <string>Next File</string>
   </property>
   <property name="shortcut">
    <string>PgDown</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="icon">
    <iconset theme="go-previous">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Previous File</string>
   </property>
   <property name="toolTip">
    <string>Previous File</string>
   </property>
   <property name="shortcut">
    <string>PgUp</string>
   </property>
  </action>
  <action name="actionOriginalSize">
   <property name="icon">
    <iconset theme="zoom-original">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Original Size</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="actionZoomFit">
   <property name="icon">
    <iconset theme="zoom-fit-best">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Fit</string>
   </property>
  </action>
  <action name="actionRotateClockwise">
   <property name="icon">
    <iconset theme="object-rotate-right">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Rotate Clockwise</string>
   </property>
  </action>
  <action name="actionRotateCounterclockwise">
   <property name="icon">
    <iconset theme="object-rotate-left">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Rotate &amp;Counterclockwise</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>P&amp;references</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>&amp;Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionFirst">
   <property name="icon">
    <iconset theme="go-first">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>First File</string>
   </property>
   <property name="shortcut">
    <string>Home</string>
   </property>
  </action>
  <action name="actionLast">
   <property name="icon">
    <iconset theme="go-last">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Last File</string>
   </property>
   <property name="shortcut">
    <string>End</string>
   </property>
  </action>
  <action name="actionNewWindow">
   <property name="icon">
    <iconset theme="document-new">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;New Window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionFlipHorizontal">
   <property name="text">
    <string>Flip &amp;Horizontally</string>
   </property>
  </action>
  <action name="actionScreenshot">
   <property name="icon">
    <iconset theme="camera-photo">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Capture Screenshot</string>
   </property>
  </action>
  <action name="actionFullScreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>F&amp;ull Screen</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
  </action>
  <action name="actionFlipVertical">
   <property name="text">
    <string>Flip &amp;Vertically</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset theme="edit-paste">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Paste from Clipboard</string>
   </property>
  </action>
  <action name="actionSlideShow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="media-playback-start">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Slide Show</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset theme="edit-delete">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionShowThumbnails">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Thumbnails</string>
   </property>
  </action>
  <action name="actionFileProperties">
   <property name="text">
    <string>File Properties</string>
   </property>
  </action>
  <action name="actionOpenDirectory">
   <property name="icon">
    <iconset theme="document-open"/>
   </property>
   <property name="text">
    <string>Open &amp;Directory</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionUpload">
   <property name="icon">
    <iconset theme="upload-media"/>
   </property>
   <property name="text">
    <string>Upload</string>
   </property>
   <property name="toolTip">
    <string>Upload the image</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LxImage::ImageView</class>
   <extends>QGraphicsView</extends>
   <header>imageview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
